packages/debian/Dockerfile

```diff
@@ -11,8 +11,8 @@ RUN apt update -y \
         devscripts \
         build-essential \
         lintian \
-        apt-rdepends \
```

Comment:
Ha good catch

Forgot to remove this one when I rewrite the download in Python.

Reply:
I commit this change in a separate commit ;)